src/main/java/read/write/file/FileWriterImpl.java

```diff
@@ -0,0 +1,16 @@
+package read.write.file;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String content, String filePath) {
+        try {
+            Files.write(Paths.get(filePath), content.getBytes());
+        } catch (IOException e) {
+            throw new RuntimeException("Error writing file", e);
+        }
+    }
```

Comment:
Consider using try-with-resources to ensure that all file resources are properly closed after the operation is completed.